asdf_astropy/converters/transform/tabular.py

```diff
@@ -0,0 +1,52 @@
+from .core import TransformConverterBase
+
+
+__all__ = ["TabularConverter"]
+
+
+class TabularConverter(TransformConverterBase):
+    """
+    ASDF support for serializing tabular models.
+    """
+    tags = ["tag:stsci.edu:asdf/transform/tabular-*"]
+
+    types = [
+        "astropy.modeling.tabular.Tabular1D",
+        "astropy.modeling.tabular.Tabular2D",
+    ]
+
+    def to_yaml_tree_transform(self, model, tag, ctx):
+        node = {}
+        node["fill_value"] = model.fill_value
+        node["lookup_table"] = model.lookup_table
+        node["points"] = [p for p in model.points]
+        node["method"] = str(model.method)
+        node["bounds_error"] = model.bounds_error
+        return node
+
+    def from_yaml_tree_transform(self, node, tag, ctx):
+        from astropy.modeling import tabular
+
+        lookup_table = node.pop("lookup_table")
+        dim = lookup_table.ndim
+        fill_value = node.pop("fill_value", None)
+        if dim == 1:
+            # TODO(eslavich): What does this mean?  [:] merely creates a view
+            # over the same memory-mapped array.
+            # The copy is necessary because the array is memory mapped.
+            points = (node["points"][0][:],)
```

Comment:
That's a good question.  Would `points = (node["points"][0],)` work?

Reply:
I don't see why not...  maybe the comment is leftover from some previous implementation?

Reply:
So it used to be this

https://github.com/asdf-format/asdf/pull/220/files

Reply:
And it was changed here

https://github.com/asdf-format/asdf/pull/222/files

with no change of the comment.

Reply:
I'll drop the indexing and remove the comment, but @nden let me know if there's a reason why I should revive it.